EDEngineer/Resources/Data/blueprints.json

```diff
@@ -16406,5 +16406,65 @@
       }
     ],
     "Grade": 5
+  },
+  {
+    "Type": "The Sarge",
+    "Name": "unlock",
+    "Engineers": [ "@Unlock" ],

```

Comment:
synthesis basically means the player himself ; the main idea is to have a dedicated filter

with engineer = The Sarge, users will be able to see everything related to The Sarge by filtering blueprints with his name ; and in the Type filters, they will have the "Unlock" type as well